src/sp_disabled_functions.c

```diff
@@ -213,12 +213,49 @@ static bool is_param_matching(zend_execute_data* execute_data,
   return false;
 }
 
+static zend_execute_data* is_file_matching(
+    zend_execute_data* const execute_data,
+    sp_disabled_function const* const config_node,
+    char const* const current_filename) {
+#define ITERATE(ex)                                            \
```

Comment:
better check that ex != NULL before `ex = ex->prev`, smells null deref in weird context (ie. ob_start etc)

Reply:
This check should be added right at the beginning on  `should_disable` in my opinion.